samples/hello_world/src/main.c

```diff
@@ -6,8 +6,243 @@
 
```

Comment:
You need to move this to its own sample or something, not modify hello_world

Reply:
OK. I'll remove test code from hello_world.